src/core/queries.tsx

```diff
@@ -15,7 +16,7 @@ interface LoadMore<TData, TVariables> {
 }
 
 interface TypedQueryInnerProps<TData, TVariables> {
-  children: (
+  children?: (
```

Comment:
Why would you use query without children to pass data to?

Reply:
To use query component instead of doing it manually 
https://github.com/mirumee/saleor-storefront/pull/205/files#diff-83953a0f45aa4e2f549df5bc06506c43R25

Reply:
In this particular case, y not just use
```
<Query>
  {({data}) => <Provider value={data} />}
</Query>
```

This way it's clean, declarative and stateless.